src/libraries/System.Text.Json/System.Text.Json.SourceGeneration.Tests/JsonSourceGeneratorTests.cs

```diff
@@ -57,6 +57,24 @@ public class IndexViewModel
         public bool HasFeaturedCampaign => FeaturedCampaign != null;
     }
 
+    [JsonSerializable]
+    public class WeatherForecastWithPOCOs
+    {
+        public DateTimeOffset Date { get; set; }
+        public int TemperatureCelsius { get; set; }
+        public string Summary { get; set; }
+        public string SummaryField;
+        public List<DateTimeOffset> DatesAvailable { get; set; }
+        public Dictionary<string, HighLowTemps> TemperatureRanges { get; set; }
+        public string[] SummaryWords { get; set; }
+    }
+
+    public class HighLowTemps
+    {
+        public int High { get; set; }
+        public int Low { get; set; }
+    }
+
     public class JsonSerializerSourceGeneratorTests
```

Comment:
I think this class should be static + all test members?

Reply:
updated, my bad